webapp/src/views/ServiceList.vue

```diff
@@ -24,44 +35,107 @@
           <th scope="col">Vorname</th>
           <th scope="col">Nachname</th>
           <th scope="col">Datum</th>
+          <th scope="col">Kommentar</th>
         </tr>
       </thead>
       <tbody>
         <tr
           v-for="(person, index) in serviceHistory"
           :key="index"
           :class="{
-            requested: person.groupMemberStatus === 'Requested',
-            todelete: person.groupMemberStatus === 'ToDelete',
+            requested: person.groups[0].groupMemberStatus === 'Requested',
+            todelete: person.groups[0].groupMemberStatus === 'ToDelete',
           }"
         >
           <th scope="row">{{ index }}</th>
           <td>{{ person.firstName }}</td>
           <td>{{ person.lastName }}</td>
-          <td>{{ person.serviceDates.join(", ") }}</td>
+          <td>
+            {{ person.serviceDates.map((ele) => ele.date).join(", ") }}
+          </td>
+          <td>{{ person.groups[0].comment }}</td>
```

Comment:
In this case I would prefer a cheap solution: Prepend the group name and join with comma, e.g. _Kaffeedienst: erstmal Pause, Kehren am Samstag: gerne auch zweimal im Jahr_.